### PR TITLE
Chore: Update project metadata, fix release tooling

### PR DESCRIPTION
**Pull request by amotl**
Hi again,

just some minor updates before running a new release.

With kind regards,
Andreas.

### CODE REVIEW

File: crate_ruby.gemspec [hunk 1]
```diff
@@ -29,16 +29,32 @@ Gem::Specification.new do |spec|
...
-  spec.description   = 'A Ruby interface for Crate.IO. Put your data to work. Simply.'
-  spec.homepage      = 'http://crate.io'
+  spec.summary       = 'CrateDB HTTP client library for Ruby'
+  spec.description   = 'A Ruby interface for CrateDB featuring query support, DDL command shortcuts, and support for BLOB tables'
+  spec.homepage      = 'https://crate.io'
   spec.license       = 'Apache-2.0'
   spec.required_ruby_version = '>= 2.6'
 
-  spec.files         = `git ls-files -z`.split("\x0")
+  spec.files         = Dir['lib/**/*']
```
Comment (amotl) [inline crate_ruby.gemspec:38]: Packaging the whole repository content is crazy, let's get sane again.

File: crate_ruby.gemspec [hunk 2]
```diff
@@ -29,16 +29,32 @@ Gem::Specification.new do |spec|
...
+    '--quiet'
+  ]
+
+  spec.metadata = {
+    'bug_tracker_uri' => 'https://github.com/crate/crate_ruby/issues',
+    'changelog_uri' => 'https://github.com/crate/crate_ruby/blob/main/CHANGES.rst',
+    'documentation_uri' => 'https://www.rubydoc.info/gems/crate_ruby/0.2.0',
+    'homepage_uri' => spec.homepage,
+    'source_code_uri' => 'https://github.com/crate/crate_ruby'
+  }
```
Comment (amotl) [inline crate_ruby.gemspec:57]: Inspired by https://piotrmurach.com/articles/writing-a-ruby-gem-specification/.

File: crate_ruby.gemspec [hunk 3]
```diff
@@ -27,18 +27,34 @@ require 'crate_ruby/version'
...
+    '--main', 'README.rst',
+    '--line-numbers',
+    '--inline-source',
+    '--quiet'
+  ]
+
+  spec.metadata = {
+    'bug_tracker_uri' => 'https://github.com/crate/crate_ruby/issues',
+    'changelog_uri' => 'https://github.com/crate/crate_ruby/blob/main/CHANGES.rst',
+    'documentation_uri' => 'https://www.rubydoc.info/gems/crate_ruby/0.2.0',
```
Comment (hammerhead) [inline crate_ruby.gemspec:54]: Can we use `CrateRuby::VERSION` here to prevent the hardcoded version?

**Review by hammerhead — APPROVED** [review]

File: crate_ruby.gemspec (hunk 3 above)
Comment (amotl) [reply crate_ruby.gemspec]: Thanks for spotting this. Let's just omit the the version number here. Fixed with 3fade6ef.

**Pull request merged**